compose/config/config.py

```diff
@@ -213,6 +214,9 @@ def find(base_dir, filenames):
             os.getcwd(),
             [ConfigFile(None, yaml.safe_load(sys.stdin))])
 
+    if filename.startswith('./http'):
```

Comment:
Actually, reference to the build system of Go or Java, I think we can simply support HTTP/HTTPS. Because they're much more popular them other protocols. curl http://stuff/to/download | docker-compose -f - is just a little bit long. And you know what, the container version of docker-compose cannot even read stdin properly. I think that's a bug but this can be a reason for us to consider the improvement.